DotaStats/AppCoordinator.swift

```diff
@@ -28,25 +28,19 @@ final class AppCoordinator {
         setupNavigationBarAppereance()
     }
 
-    private func createNavigationController(for viewController: UIViewController,
-                                            with title: String) -> UINavigationController {
+    private func makeNavigationController(rootViewController viewController: UIViewController,
```

Comment:
Кажется, что внутреннее имя лишнее
```suggestion
    private func makeNavigationController(rootViewController: UIViewController,
```